app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,18 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by id: session[:user_id]
```

Comment:
toán tử `||=` có tác dụng gì?

Reply:
Tác dụng của ||= là nếu biến @current_user đã tồn tại thì lấy luôn giá trị đó còn nếu không mới gọi vào db để tìm kiếm dữ liệu. Sử dụng dấu ' ||= ' trong những trường hợp này sẽ giúp tối ưu code cũng như cải thiện performance của hệ thống